src/data/sql-quiz.ts

```diff
@@ -576,6 +576,17 @@ const sqlQuiz = [
       "The ROUND() function takes in two parameters(number and decimals) and rounds the number to the number of decimals specified in the 2nd parameter.",
     Link: "https://dev.mysql.com/doc/refman/5.7/en/precision-math-rounding.html",
   },
+  {
+    Question:
+      "In SQL, What does TCL stand for ",
```

Comment:
```suggestion
      "In SQL, what does TCL stand for?",
```

Reply:
> ```suggestion
>       "In SQL, what does TCL stand for?",
> ```

I will do the suggested changes soon